src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java

```diff
@@ -26,11 +35,51 @@
      */
     @PostMapping("/extractMetadata")
     public String uploadFile(@RequestParam("file") MultipartFile file) throws IllegalStateException, IOException {
+        LOGGER.info("start");
         String filename = file.getOriginalFilename();
         String storagePath = fileStorageService.uploadFile(file);
         String outputPath = FileStorageService.getOutputPath(filename);
         final String baseUrl = ServletUriComponentsBuilder.fromCurrentContextPath().build().toUriString();
-        String downloadLink = baseUrl + "/api/download/" +  filename;
+        String downloadLink = baseUrl + "/api/download/" + filename;
+        LOGGER.info("finish");
         return downloadLink;
     }
+
+    /**
+     * Endpoint for extract metadata
+     */
+    @GetMapping("/metadata")
+    public ResponseEntity<String> extractMetadataDefaultName() {
+        LOGGER.info("start");
+
+        File fileToExtract = new File("C:\\Users\\ASUS\\Desktop\\AT Materias\\Prog102\\prog102-AT13-JalaConverter\\JalaConverter\\Images\\img7.jpg");
+        File fileToExport = new File("C:\\Users\\ASUS\\Desktop\\AT Materias\\Prog102\\prog102-AT13-JalaConverter\\JalaConverter\\archive");
+
+        ExtractMetadata extractMetadata = new ExtractMetadata(fileToExtract, fileToExport);
+        extractMetadata.extractMetadata();
+        LOGGER.info("finish");

```

Comment:
these spaces in the code should be removed

Reply:
ok